### PR TITLE
New pattern fix for MasterPage creation

### LINKED ISSUE #140: Apply the new pattern to create MasterPage
A new pattern to create MasterPage was discussed in https://github.com/Starcounter/Guidelines/issues/23 and implemented for reference in https://github.com/StarcounterApps/Website/pull/106.

Please apply the same pattern in KitchenSink app, namely:

- replace `Handle.GET("/KitchenSink/master"` handler with a `GetSessionFromMaster` method

### PR DESCRIPTION
**Pull request by bouguila**
Fix #140
Replaced handle.GET("kitchensink/master" with GetMasterFrom Session

### CODE REVIEW

File: src/KitchenSink/DropdownPage.json.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ protected override void OnData()
 
         public string SelectedProductKeyBind
         {
-            get => SelectedProduct?.Key;
+            get { return SelectedProduct?.Key; }
```
Comment (PatrykSzwer) [inline src/KitchenSink/DropdownPage.json.cs]: I'm confused, why this change happen? Old solution was clearer and shorter.

Comment (bouguila) [inline src/KitchenSink/DropdownPage.json.cs]: for some strange  reason, old solution it wasn't compiling on my local machine.. i will investigate on it again

Comment (PatrykSzwer) [inline src/KitchenSink/DropdownPage.json.cs]: It's feature of C# 7. Are you using Visual studio 2017?

Comment (bouguila) [inline src/KitchenSink/DropdownPage.json.cs]: i was using vs15 untill last week, now i have vs17 and seems to be working ok. thanks

File: src/KitchenSink/Program.cs [hunk 2]
```diff
@@ -283,15 +263,23 @@ static void Main()
             return master;
         }
 
-        private static int IndexOfNth(string str, char c, int n)
```
Comment (PatrykSzwer) [inline src/KitchenSink/Program.cs]: Great! Thx for that.

Comment (bouguila) [inline src/KitchenSink/Program.cs]: perfect! thanks for the review. merging now.
